MAINTAINERS.md

```diff
@@ -1,10 +1,13 @@
+# OpenAPI Specification Editors
+
 ## Active
 * Darrel Miller [@darrelmiller](https://github.com/darrelmiller)
 * Jeremy Whitlock [@whitlockjc](https://github.com/whitlockjc)
 * Marsh Gardiner [@earth2marsh](https://github.com/earth2marsh)
 * Ron Ratovsky [@webron](https://github.com/webron)
 
-## Provisional
```

Comment:
Provisional does refer back to the governance processes. What do we lose by dropping it here?